guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -7,7 +7,7 @@ To discontinue using Puppet in your {Project}, follow this procedure.
 * Puppet is enabled on {Project}.
 
 .Procedure
-. If you used Puppet server on external {SmartProxies}, disable Puppet server on all external {SmartProxies}:
+. If you used Puppet server on any {SmartProxies}, disable Puppet server on all {SmartProxies}:
```

Comment:
Perhaps this is an influence of how you'd say it in Dutch, but I feel like something's missing here. Isn't it either "If you use Puppet server" or "If you've used Puppet server"?